aws/provider.go

```diff
@@ -438,6 +438,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_network_interface_sg_attachment":          resourceAwsNetworkInterfaceSGAttachment(),
 			"aws_default_security_group":                   resourceAwsDefaultSecurityGroup(),
 			"aws_security_group_rule":                      resourceAwsSecurityGroupRule(),
+			"aws_servicecatalog_portfolio":                 resourceAwsServiceCatalogPortfolio(),
```

Comment:
Can you also add the documentation for this resource?

Reply:
@Ninir will do